assets/src/data/content/resource.ts

```diff
@@ -180,6 +180,7 @@ export const PurposeTypes: Purpose[] = [
   { value: 'quiz', label: 'Quiz' },
   { value: 'simulation', label: 'Simulation' },
   { value: 'walkthrough', label: 'Walkthrough' },
+  { value: 'priorknowledgecheck', label: 'Prior Knowledge Check' },
```

Comment:
NIT: WDYT about adding this option after the "Many students wonder"'s option to keep them sorted alphabetically?

Reply:
I just pushed your changes thnks